cmd/api/src/config/default.go

```diff
@@ -36,7 +36,6 @@ func NewDefaultConfiguration() (Configuration, error) {
 		return Configuration{
 			Version:                      0,
 			BindAddress:                  "127.0.0.1",
-			NetTimeoutSeconds:            70,  // Default timeout to avoid race conditions with 60 second gateway timeouts
```

Comment:
Are we just getting rid of this configuration element? It looks like all of the potential wire-up for it is now removed as per this changeset.

If the intent is to remove this configuration element we should follow up and review relevant documentation or other mention of it.

Reply:
Think we found one last place this is mentioned but I'm going to go ahead and mark this resolved :+1: